README.md

```diff
@@ -1 +1,31 @@
-# Dinner-with-death-project
+# Dinner With Death
+
+## Why?
+I like text adventure games. As well as write stories in my free time. So it felt like the most direct path for an app to build. In conjunction I also wanted something easy. I didn't want to get lost in the sauce and be unable to finish because I wasn't up to par. Turns out however that I greatly underestimated myself and now I'm left with a project I can't wait to get back to and increase the size of.
```

Comment:
Suggested grammar/punctuation fixes:
I like text adventure games. As well as write stories in my free time. So it felt like the most direct path for an app to build. In conjunction, I also wanted something easy. I didn't want to get lost in the sauce and be unable to finish because I wasn't up to par. However, it turns out that I greatly underestimated myself, and now I'm left with a project I can't wait to get back to and increase the size of.